app/models/concerns/publishable.rb

```diff
@@ -14,27 +14,25 @@ module Publishable
 
   def publish!
     publish_content_item!
-    publish_rummager_artefact!
+    add_to_search_index!
+  end
+
+  def base_path
+    "/government/policies/#{slug}"
```

Comment:
The email signup work is on hold for now, so if the concern is going away either before or during that work I'll merge this as-is.